packages/components/forma-36-react-datepicker/tsconfig.json

```diff
@@ -20,8 +20,8 @@
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
+    "allowJs": true,
```

Comment:
Is there a reason to add this?

Reply:
Oh I forgot now, but I got an error when building the datepicker and the error message suggested to set `allowJs` to true
I will remove this and get the error again so we can analyze this better because I forgot hahahah

Reply:
Good news! I removed this and no errors! hahahaha